packages/core-mobile/app/hooks/earn/useIssueDelegation.ts

```diff
@@ -55,6 +64,8 @@ export const useIssueDelegation = (
       stakingAmount: TokenUnit
       startDate: Date
       endDate: Date
+      feeState?: pvm.FeeState
```

Comment:
I think you should pass gasPrice instead here and then call getFeeState(gasPrice) inside here